src/libraries/System.Private.CoreLib/src/System/String.Manipulation.cs

```diff
@@ -1597,6 +1614,13 @@ private void MakeSeparatorList(ReadOnlySpan<char> separators, ref ValueListBuild
                     sep0 = separators[0];
                     sep1 = separators[1];
                     sep2 = separators[2];
+
+                    if (Avx2.IsSupported && 16 <= Length)
```

Comment:
Nit, ` 16 <= Length` is [Yoda style](https://en.wikipedia.org/wiki/Yoda_conditions) which we don't normally use in C#.

Reply:
Fixed in https://github.com/dotnet/runtime/pull/38001/commits/65cba0ca5ccc2bb55fa8044cb7326e2e0b6a455a